### DISCUSSION TITLE
Geodesic Community Group

### DISCUSSION THREAD

**Discussion by rastoddart** [geodesicsolutions-community/geocore-community#185] [Ideas]
Greetings from Lodi, CA.

I would like to assist "imxweb" in creating/maintaining a place on the WWW for people using the Geodesic script. If you are interested, my email address is public in my profile for the next week. Email me with Geodesic as the Subject of your email and let me know that you are in fact interested and we will see what develops. If you have contact with other persons using the script, please pass this information along to them.

Thank you,

Russ

**Comment by jonyo** [comment]
I am out of a personal computer for the next few weeks, my personal laptop's battery went out (it even expanded, yikes!).  Instead of just repairing it I opted to order a desktop since I hardly used it as a laptop 😂..  Which may be part of why it went out faster than it should, always being plugged in...

Anyways so I can't do much until the new computer gets here, current estimate is 2-3 weeks.  Just wanted to mention so you are aware, it will be difficult to do much until then, like find a way to copy the wiki or things like that.

**Comment by rastoddart** [comment]
As a follow-up, I received a few thoughtful email replies. It seems to me that a dedicated website/forum for users of the script would be beneficial to the Geo community and serve as a resource for anyone looking for a developer, addon's etc. Additionally, the ability to coordinate with other like minded users and raise funds for maintaining/development/upgrades etc. might extend the life of the script itself. For the time being, it appears that this sort of resource is not necessary.
